internal/testutil/acc/cluster.go

```diff
@@ -102,3 +67,55 @@ func ExistingClusterUsed() bool {
 	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
 	return clusterName != "" && projectID != ""
 }
+
+type ReplicationSpecRequest struct {
+	ZoneName     string
+	Region       string
+	InstanceSize string
+	ProviderName string
+	NodeCount    int
+}
+
+func (r *ReplicationSpecRequest) AddDefaults() {
+	if r.NodeCount == 0 {
+		r.NodeCount = 3
+	}
+	if r.ZoneName == "" {
+		r.ZoneName = "Zone 1"
+	}
+	if r.Region == "" {
+		r.Region = "US_WEST_2"
+	}
+	if r.InstanceSize == "" {
+		r.InstanceSize = "M10"
+	}
+	if r.ProviderName == "" {
+		r.ProviderName = constant.AWS
+	}
+}
+
+func ReplicationSpec(req *ReplicationSpecRequest) admin.ReplicationSpec {
+	if req == nil {
+		req = new(ReplicationSpecRequest)
+	}
+	req.AddDefaults()
+	defaultNumShards := 1
```

Comment:
this feels like should belong to AddDefaults but i guess it's not in ReplicationSpecRequest

Reply:
Yupp